src/common/formatters.js

```diff
@@ -0,0 +1,4 @@
+export const formatScoreTime = (timeInSeconds) => {
```

Comment:
I'd place it to `shared` folder and reuse in countdown components, as it formats time in the same manner

Reply:
done